package.json

```diff
@@ -14,8 +14,8 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "pretest": "jshint .",
-    "test": "mocha -R spec --timeout 60000"
+    "test": "mocha -R spec --timeout 60000",
+    "posttest": "eslint --ignore-path .gitignore ."
```

Comment:
Both `pretest` and `posttest` has pros and cons. As @superkhau wrote, we decided to use `posttest` in LoopBack. Let's not waste our time opening this discussion again, there is ton of more important work to focus on.